src/app/series/directives/series-podcast.component.css

```diff
@@ -25,3 +25,6 @@
 .feed-url button:hover, .feed-url button:focus {
   background-color: #e28b42;
 }
+input[type=checkbox] {
+  vertical-align: middle;
+}
```

Comment:
noting that `vertical-align: middle` should probably be our default on checkboxes since we use it wherever we use checkboxes, one for the styleguide!